packages/storybook8/stories/Feedback.mdx

```diff
@@ -7,8 +7,16 @@ import { addCSS } from './utils';
 <div id='feedback-page'>
 <script> { 
   addCSS(' '.concat(
-  '#feedback-page span { font-family: Segoe UI, sans-serif; color: white; font-weight: 600; font-size: 0.875rem; }'
-)) } </script>
+  '#feedback-page span { font-family: Segoe UI, sans-serif; color: white; font-weight: 600; font-size: 0.875rem; margin: 0px 4px; }'
```

Comment:
Is this how we do custom CSS for a story in SB8?

Reply:
ideally we don't do any custom styling. I just noticed there was a different in the height property that we don't have to deal with in SB6.